SecretSanta/src/SecretSanta.Business/Gift.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace SecretSanta.Business
+{
+    public class Gift
+    {
+        private readonly int _Id = 0;
```

Comment:
The default value of an `int` is zero, so the setter is redundant.
```suggestion
        private readonly int _Id;
```